src/App.js

```diff
@@ -1,25 +1,51 @@
-import logo from './logo.svg';
+import { useEffect } from 'react';
+import { useState } from 'react';
 import './App.css';
+import DogImage from './DogImage';
+import Dropdown from './Dropdown';
+
 
 function App() {
+
+  const [dogUrl, setDogUrl]= useState(''); 
+  const [breed, setBreed]= useState('');
+  const [breedName, setBreedName]= useState('');
+
+
```

Comment:
Perhaps implementing some pseudo code to let other developers know the thought process and a footer! 🐶